source/_integrations/nmbs.markdown

```diff
@@ -1,6 +1,6 @@
 ---
-title: NMBS
-description: Instructions on how to integrate timetable data for traveling on the NMBS Belgian Railway within Home Assistant.
+title: NMBS/SNCB
```

Comment:
This is the name of the integration, thus this need to be adjusted in the Home Assistant code-base to match this.

Have you opened a PR to adjust the integration manifest?

Reply:
Let's not do the title change then. Description and fix typo should be enough?

Reply:
Sounds good to me 👍